tests/regression/lp-1867193/task.yaml

```diff
@@ -0,0 +1,24 @@
+summary: certain refresh sequence on the maas snap breaks the layout system
+systems: [ubuntu-18.04-64] # tight coupling with container guest
+prepare: |
```

Comment:
Nitpick, empty new lines between test sections improve overall readability.

Reply:
I don't know, it fits on a (small) screen. I have no strong feeling either way.